webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './AnsibleVariableOverrides';
```

Comment:
I think in https://community.theforeman.org/t/rethinking-react-redux-folder-structure/24183 we agreed that the main component should be directly in the index.js file?